app/src/main/admin/components/MainMenu/MainMenuNew.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
```

Comment:
What's stopping us replacing the old `MainMenu` component with this new one?

Reply:
it will not work with users part, as breadcrumbs are using alt there and will conflict with parent page